README.md

```diff
@@ -44,13 +44,15 @@ more files to this list as you like:
 ```rust,no_run
 extern crate skeptic;
 
+use std::path::PathBuf;
```

Comment:
you will be able to drop this change if you use `into` instead of `from` as suggested below